src/applications/personalization/profile/components/ProfileInformationEditView.jsx

```diff
@@ -95,13 +94,6 @@ export class ProfileInformationEditView extends Component {
     ) {
       window.clearInterval(this.interval);
     }
-    // if a transaction was created that was immediately successful (for example
-    // when the transaction's status is `COMPLETED_NO_CHANGES_DETECTED`),
-    // immediately exit edit view and clear the transaction request so it can be triggered again
-    if (isSuccessfulTransaction(this.props.transaction)) {
-      this.props.openModal(null);
-      this.props.clearTransactionRequest(this.props.fieldName);
-    }
```

Comment:
Removed bc now the transaction is treated as successful if no changes are detected